nh_eobs/wizard/print_observation_report.py

```diff
@@ -68,8 +69,12 @@ def print_report(self, cr, uid, ids, context=None):
             'name': 'nh.clinical.observation_report',
             'datas': base64.encodestring(report_pdf),
             'datas_fname': '{nhs}_{date}_observation_report.pdf'.format(
-                nhs=patient_nhs, date=datetime.strftime(
-                    datetime.now(), '%Y%m%d')),
+                nhs=patient_nhs,
+                date=datetime.strftime(
+                    date_utils_pool.get_localised_time(),
+                    '%Y%m%d'
+                )
+            ),
```

Comment:
Are there any stories or acceptance criteria for the report name? It could be quite important so this might be one of those times where its worth checking and putting something in place if we don't have it. Even if its just:

1. Quick chat in Slack about AC.
2. Slap together a story with only one or two scenarios in it.
3. Write a couple of tests.

I recommend this partly because its good to backfill missing requirements but also because then you'll know for sure that these changes haven't changed behaviour (even though they look mundane enough you never know).

Reply:
True dat. We have a story for the BCP not the printed report. We could use the need to have the Spell ID in the 'adhoc' report filename so you can see which spell for the patient it relates to

Reply:
Created an analysis task for the correct name but also will add tests around this call to raise coverage on report